scripts/uberenv/packages/axom/package.py

```diff
@@ -51,7 +51,7 @@ class Axom(CMakePackage, CudaPackage):
     homepage = "https://github.com/LLNL/axom"
     git      = "https://github.com/LLNL/axom.git"
 
-    version('master', branch='master', submodules=True)
+    version('main', branch='main', submodules=True)
```

Comment:
These changes are the from the new Axom package...

Reply:
I was corrected in a Spack PR when changing "master" for "main" in the first argument: "master" is a specific value for spack. In order to preserve the original behavior (master is always newer that a release), then the correct syntax is:
```suggestion
    version('master', branch='main', submodules=True)
```

Reply:
That’s definitely messy, and I hope they will soon put "main" at the same level as "master".

Reply:
I actually got a fix from greg in one of my PR's to add main as a known branch in Spack.So it should be good now.